tests/test_workers/test_tasks/test_opm_operations.py

```diff
@@ -155,3 +156,214 @@ def test_serve_cmd_at_port_delayed_initialize(
     cmd = ['opm', 'registry', 'serve', '-p', '50051', '-d', '/tmp/dummy.db', '-t', '/dev/null']
     opm_operations._serve_cmd_at_port(" ".join(cmd), '/tmp', 50051, 5, 3)
     assert mock_run_cmd.call_count == 7
+
+
+@mock.patch('iib.workers.tasks.opm_operations.shutil.rmtree')
+@mock.patch('iib.workers.tasks.utils.run_cmd')
+def test_opm_migrate(
```

Comment:
This belongs to different commit - I guess it will be cleaned once properly we rebase our code.

Reply:
yup, that's right